src/test/java/uk/gov/pay/adminusers/utils/email/EmailValidatorIsPublicSectorEmailTest.java

```diff
@@ -109,7 +110,8 @@ public EmailValidatorIsPublicSectorEmailTest(String email, boolean testResult) {
                 {"valid@subdomain.scotent.co.uk", true},
                 {"valid@subdomain.slc.co.uk", true},
                 {"valid@subdomain.socialworkengland.org.uk", true},
-                {"valid@subdomain.ucds.email", true}
+                {"valid@subdomain.ucds.email", true},
+                {"valid@subdomain.wmca.org.uk", true},
```

Comment:
May be you don't need the `,` here, but actually it is easier to copy and paste in the future